Source/Fuse.Common/Timer.uno

```diff
@@ -20,17 +20,23 @@ namespace Fuse
 			_once = true;
 		}
 
-		void Start()
+		public void Start()
 		{
-			_startTime = Uno.Diagnostics.Clock.GetSeconds();
-			UpdateManager.AddAction(Update);
-			_running = true;
+			if(!_running)
+			{
+				_startTime = Uno.Diagnostics.Clock.GetSeconds();
+				UpdateManager.AddAction(Update);
+				_running = true;
+			}
 		}
 
-		void Stop()
+		public void Stop()
 		{
-			_running = false;
-			UpdateManager.RemoveAction(Update);
+			if(_running) 
+			{
+				_running = false;
+				UpdateManager.RemoveAction(Update);
+			}
 		}
```

Comment:
Rather than doing this, I suggest you make this IDisposable, and make the dispose-method call `Stop()`. That generalizes the interface a bit, down to only the things you need to know about.